### PR TITLE
ODC-7654: Remove duplicate Operator checks from Shipwright package

### PR DESCRIPTION
**Pull request by The-Anton**
Story: [ODC-7654](https://issues.redhat.com/browse/ODC-7654)

### CODE REVIEW

File: frontend/packages/shipwright-plugin/integration-tests/support/commands/hooks.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  /* returning false here prevents Cypress from failing the test */
+  if (resizeObserverLoopErrRe.test(err.message)) {
+    return false;
+  }
+  cy.log('uncaught:exception', err, runnable, promise);
+});
+
+before(() => {
+  cy.exec('../../../../contrib/create-user.sh');
+  cy.login();
```
Comment (sanketpathak) [inline frontend/packages/shipwright-plugin/integration-tests/support/commands/hooks.ts:17]: This will log in as kube:admin user

**Comment by sanketpathak** [comment]
Login through non-admin user will be addressed with a different pr 
/lgtm
/approve

**Comment by The-Anton** [comment]
The pr only edits e2e tests
/label docs-approved
/label px-approved
/label qe-approved

**Pull request merged**